crates/ty_server/tests/e2e/main.rs

```diff
@@ -86,11 +110,7 @@ impl TestServerError {
 
 /// A test server for the ty language server that provides helpers for sending requests,
 /// correlating responses, and handling notifications.
-///
-/// The [`Drop`] implementation ensures that the server is shut down gracefully using the described
-/// protocol in the LSP specification. It also ensures that all messages sent by the server have
-/// been handled by the test client before the server is dropped.
-pub(crate) struct TestServer {
+pub struct TestServer {
```

Comment:
Does the `TestServer` need to be public. Aren't all tests in the same package (I don't know)

Reply:
No, it does not, was a leftover from previous version. I've made it `pub(crate)`.